docs/guides/4f-js-rulesets.md

```diff
@@ -0,0 +1,110 @@
+## JavaScript Ruleset Format
+
+Spectral v6.0 added support for a JavaScript ruleset format, similar to the JSON and YAML formats.
+
+This has a few benefits: it lets you explicitly load formats or rulesets to get control over versioning, you can load common functions from popular JS libraries, and in general feels a lot more welcoming to developers experienced with JavaScript, especially when it comes to working with custom functions.
+
+**Example**
+
+To create a JavaScript ruleset, the first step is creating a folder. In your terminal, run the following commands:
+
+```
+mkdir style-guide
+cd style-guide
+```
+
+Next, install two dependencies using [npm](https://www.npmjs.com/):
+
+```
+npm install --save @stoplight/spectral-functions
+npm install --save @stoplight/spectral-formats
+```
+
+Installing these packages is not required for creating a JavaScript ruleset, but we'll use them in our example to create some common rules used with Spectral and to target a specific OpenAPI format.
+
+Next, let's create a JavaScript file to hold our ruleset:
+
+```
+touch spectral.js
+```
+
+And inside the file, let's create a couple rules:
```

Comment:
I've seen the word "let's" a few times. Not a huge fan and it breaks our Vale rule. You could just remove it and say, "Inside the file, create a couple of rules:"

You do need the "of" between couple and rules, though.